model/command_args.go

```diff
@@ -11,18 +11,21 @@ import (
 )
 
 type CommandArgs struct {
-	UserId          string               `json:"user_id"`
-	ChannelId       string               `json:"channel_id"`
-	TeamId          string               `json:"team_id"`
-	RootId          string               `json:"root_id"`
-	ParentId        string               `json:"parent_id"`
-	TriggerId       string               `json:"trigger_id,omitempty"`
-	Command         string               `json:"command"`
-	SiteURL         string               `json:"-"`
-	T               goi18n.TranslateFunc `json:"-"`
-	Session         Session              `json:"-"`
-	UserMentions    UserMentionMap       `json:"-"`
-	ChannelMentions ChannelMentionMap    `json:"-"`
+	UserId    string               `json:"user_id"`
+	ChannelId string               `json:"channel_id"`
+	TeamId    string               `json:"team_id"`
+	RootId    string               `json:"root_id"`
+	ParentId  string               `json:"parent_id"`
+	TriggerId string               `json:"trigger_id,omitempty"`
+	Command   string               `json:"command"`
+	SiteURL   string               `json:"-"`
+	T         goi18n.TranslateFunc `json:"-"`
+
+	// DO NOT USE Session field is depricated. MM-26398
+	Session Session `json:"-"`
```

Comment:
👍 

Should we move it to the very bottom -- separate for all the valid fields?